### PR TITLE
Example capturing METS stuctural metadata

### PR DESCRIPTION
**Pull request by chmayo**
JSON-LD sample created to capture collection structure originally stored in a METS XML file

### CODE REVIEW

File: examples/example2.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+       "@type":"pcdm:Collection",
+       "@id":"http://bc.edu/bobhope",
+        "yams_desc":"descriptive metadata",
+        "pcdm:hasMember":[
+            {"@id":"http://bc.edu/bobhope/file1", "@type":"pcdm:Object", "pcdm:hasFile":
+                {
+                    "@id":"http://bc.edu/bobhope/file1.jpeg",
+                    "@type":"pcdm:File",
+                    "yams_tech":"technical metadata",
+                    "pdcm:memberOf":"http://bc.edu/bobhope/file1"
```
Comment (escowles) [inline examples/example2.json]: I think this should be:

```
"pcdm:memberOf": {"@id":"http://bc.edu/bobhope/file1"}
```

File: examples/example2.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                }
+            },
+            {"@id":"http://bc.edu/bobhope/file2", "@type":"pcdm:Object", "pcdm:hasFile":
+                {
+                    "@id":"http://bc.edu/bobhope/file2.jpeg",
+                    "@type":"pcdm:File",
+                    "yams_tech":"technical metadata",
+                    "pcdm:memberOf":"http://bc.edu/bobhope/file2"
```
Comment (escowles) [inline examples/example2.json]: Same as above.

**Comment by escowles** [comment]
This looks great -- just two small fixes to make sure the memberOf values are URIs instead of literals.

**Comment by escowles** [comment]
Thanks for the update, but I think the syntax isn't quite right.  I think it needs to have the "@id" key added.

If it helps, I generally plug JSON-LD into an [RDF converter](http://www.easyrdf.org/converter) to convert it to Turtle when I review it.  This helps catch syntax errors, and I generally find Turtle easier to understand.

**Comment by chmayo** [comment]
Right, I'm sorry, misread the comment. I blame lack of coffee. Thanks for
the suggestion; will the converter throw useful errors? I'm afraid I'm not
any more familiar with Turtle than I am with JSON, I'm primarily a Python
user.

On Mon, Jun 20, 2016 at 9:40 AM, Esmé Cowles notifications@github.com
wrote:

> Thanks for the update, but I think the syntax isn't quite right. I think
> it needs to have the "@id https://github.com/id" key added.
> 
> If it helps, I generally plug JSON-LD into an RDF converter
> http://www.easyrdf.org/converter to convert it to Turtle when I review
> it. This helps catch syntax errors, and I generally find Turtle easier to
> understand.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/duraspace/pcdm/pull/55#issuecomment-227144689, or mute
> the thread
> https://github.com/notifications/unsubscribe/APW373Co4qV-vYe34rqVgMU-90go0-Mxks5qNphGgaJpZM4I2upV
> .

## 

Christine Mayo
Digital Production Librarian
Thomas P. O'Neill, Jr. Library
Boston College
christine.mayo@bc.edu

**Comment by escowles** [comment]
No, I'm afraid it won't generate very good error messages.  You could also use the [JSON-LD Playground](http://json-ld.org/playground/) or a JSON syntax checker like [JSON Lint](http://jsonlint.com/).

**Comment by escowles** [comment]
👍

**Pull request merged**